### ISSUE TITLE
Can't install in Symfony 4.3

### ISSUE THREAD

**Issue by TomaszGasior** [kiczort/polish-validator-bundle#10]
When I try to install this bundle in my project I get error message from composer. Since my project isn't open source, I reproduced the problem with Symfony's skeleton project.

```
composer create-project symfony/website-skeleton
cd website-skeleton
composer require kiczort/polish-validator-bundle
```

Output:
```
Using version ^1.1 for kiczort/polish-validator-bundle
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "4.3.*"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - kiczort/polish-validator-bundle v1.1.0 requires symfony/framework-bundle ~2.1|~3.0 -> no matching package found.
    - kiczort/polish-validator-bundle 1.1.1 requires symfony/framework-bundle ~2.1 || ~3.0 -> no matching package found.
    - Installation request for kiczort/polish-validator-bundle ^1.1 -> satisfiable by kiczort/polish-validator-bundle[1.1.1, v1.1.0].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.
```

**Comment by mleczakm** [comment]
Please try dev-master or (released few minutes ago) version 1.2.

**Comment by TomaszGasior** [comment]
Problem fixed. Thank you.
